renovate.json5

```
@@ -0,0 +1,97 @@
+{
+	extends: [ 'config:recommended', 'default:pinDigestsDisabled', 'mergeConfidence:all-badges' ],
+	// --- Individual dependency settings ---
+	packageRules: [
+		{
+			extends: [ 'monorepo:wordpress', 'schedule:daily' ],
+			separateMajorMinor: false,
+			prPriority: 3,
+			// We want to update these ASAP, so we don't want to wait for the minimum release age.
+			minimumReleaseAge: '0 days',
+		},
+		{
+			matchDepNames: [ 'node' ],
+			prPriority: 2,
+		},
+		{
+			extends: [ 'monorepo:react', ':widenPeerDependencies' ],
+			prPriority: 2,
+			ignorePaths: [ 'packages/interpolate-components' ],
+		},
+		{
+			groupName: 'redux-related packages',
+			matchPackagePatterns: [ 'redux' ],
+			matchPackageNames: [ 'react-redux' ],
+			prPriority: 2,
+		},
+		{
+			groupName: 'typescript-related packages',
+			matchPackagePatterns: [ 'typescript-eslint' ],
+			matchPackageNames: [ 'typescript' ],
+			prPriority: 2,
+		},
+		{
+			groupName: 'webpack packages',
+			matchPackageNames: [
+				'style-loader',
+				'html-loader',
+				'exports-loader',
+				'loader-utils',
+				'postcss-loader',
+				'css-loader',
+			],
+			matchPackagePatterns: [ 'webpack', 'terser' ],
+			excludePackagePatterns: [ '^@storybook' ],
+			prPriority: 2,
+		},
+		{ extends: [ 'monorepo:storybook' ], prPriority: 2 },
+		{ extends: [ 'packages:linters' ], groupName: 'linters', prPriority: 2 },
+		{
+			extends: [ 'packages:unitTest' ],
+			groupName: 'unit test packages',
+			prPriority: 2,
+		},
+		{ extends: [ 'monorepo:babel' ], prPriority: 1 },
+		{ extends: [ 'monorepo:lodash' ], prPriority: 1 },
+		// Disable maven -- it only exists because of TeamCity, which we don't update.
+		{
+			matchManagers: [ 'maven' ],
+			enabled: false,
+		},
+	],
+	ignoreDeps: [ 'electron-builder' ],
+
+	// --- Update & schedule settings ---
+	rangeStrategy: 'bump',
+	postUpdateOptions: [ 'yarnDedupeHighest' ],
+	minimumReleaseAge: '10 days',
+	// Note that security PRs are opened immediately regardless of how many other
+	// renovate PRs are open. This keeps our queue of deps to update more manageable.
+	prConcurrentLimit: 10,
+	// Avoid overwhelming reviewers with a constant stream of updates. We'll get
+	// some new updates to handle each weekend, but throughout the week, any new
+	// PR notifications should always be actionable.
+	schedule: [ 'every weekend' ],
+	// Create PRs immediately. Extra branches we don't see still consume CI resources
+	// when they are created/rebased, so we want visibility into them quickly.
+	prCreation: 'immediate',
+	internalChecksFilter: 'strict',
+	// Don't rebase PRs constantly, to avoid using too much CI.
+	rebaseWhen: 'conflicted',
+
+	// --- Metadata settings for git ---
+	labels: [ 'Framework', '[Type] Task' ],
+	reviewers: [ 'team:team-calypso-platform' ],
```

Totally feel free to add our team as reviewers, too. Many of those packages are actually considered framework packages and we're interested to keep them up to date.

```suggestion
	reviewers: [ 'team:team-calypso-platform', 'team:team-calypso-frameworks' ],
```